Views/Layout-EmailTemplate.cshtml

```diff
@@ -0,0 +1,45 @@
+@{
+    const string blockName = "emailTemplate";
+}
+
+<!DOCTYPE html>
+<html lang="@Orchard.CultureName()">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width" initial-scale="1">
+    <!--[if !mso]>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <![endif]-->
+    <title>
+        <shape type="EmailTemplate_Title"></shape>
+    </title>
+
+    @await RenderSectionAsync("Head", false)
+
+    <shape type="EmailTemplate_LayoutInjections" prop-layoutModel="@Model"></shape>
+</head>
+<body>
+
+<span class="preheader" style="display: none; max-height: 0px; overflow: hidden;">
+    @await RenderSectionAsync("Preheader", false)

```

Comment:
Why not PreHeader (with the capital H) or BeforeHeader?

Reply:
Because this is how it's called: https://www.campaignmonitor.com/blog/email-marketing/a-practical-guide-to-email-preheaders/

Reply:
Okay, I will sic the Grammar Nazis on them.